item-api/src/main/java/com/example/auction/item/api/ItemData.java

```diff
@@ -0,0 +1,36 @@
+package com.example.auction.item.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.Builder;
+import lombok.Value;
+import lombok.experimental.Wither;
+
+import java.time.Duration;
+import java.util.UUID;
+
+
+/**
+ * Data is different from Info in that Info can be inferred from data while data is source of truth. In this
+ * case, ItemData is user generated and can't be restored or inferred from other sources.
```

Comment:
Info, as opposed to data, can be inferred from data. Data is the unique source of truth you must not loose. It is safe to loose info because you can recompute it from data. 
The error in the comment was to use `Info` (capitalised) because I was working on the idea to put `price`, `startInstant`, etc... fields on a separate class `ItemInfo`.